roles/kubernetes/master/tasks/main.yml

```diff
@@ -13,6 +13,12 @@
     dest: "{{ kube_config_dir }}/webhook-token-auth-config.yaml"
   when: kube_webhook_token_auth|default(false)
 
+- name: Create webhook authorization config
+  template:
+    src: webhook-authorization-config.yaml.j2
+    dest: "{{ kube_config_dir }}/webhook-authorization-config.yaml"
+  when: kube_webhook_authorization|default(false)
```

Comment:
This default(false) is redundant because the role defines a default. I understand the task above it is also to blame. Just pointing out a stylistic issue here.